openhands/storage/memory.py

```diff
@@ -3,12 +3,14 @@
 from openhands.core.logger import openhands_logger as logger
 from openhands.storage.files import FileStore
 
+IN_MEMORY_FILES: dict = {}
+
 
 class InMemoryFileStore(FileStore):
     files: dict[str, str]
 
     def __init__(self):
-        self.files = {}
+        self.files = IN_MEMORY_FILES
```

Comment:
Are we reasonably confident that changing this from local to global will not cause any other issues (Maybe with evaluation)?

Reply:
I think it's a reasonable change, given that it makes the behavior closer to a real filestore. Given that we only create a single filestore today (at least prior to the new storage logic) this should be safe.